files/zh-cn/mdn/writing_guidelines/page_structures/macros/commonly_used_macros/index.md

```diff
@@ -110,13 +110,6 @@ slug: MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros
   </tbody>
 </table>
 
-### 链接到某个 Bug
-
-- Bug
-
-  - 通过编号，[`bug`](https://github.com/mdn/yari/blob/main/kumascript/macros/bug.ejs) 宏可以指向 bugzilla.mozilla.org 站内相应的 bug，`\{{Bug(123456)}}` 会指向 {{Bug(123456)}}。
-  - 类似的，[`WebkitBug`](https://github.com/mdn/yari/blob/main/kumascript/macros/WebkitBug.ejs) 宏同样可以借助编号，指向 WebKit bug 库里对应的 bug。例如，`\{{WebkitBug(31277)}}` 会指向 {{WebkitBug(31277)}}。
-
```

Comment:
As we don't have any bug macros left in zh-CN. It's better to remove the documation (even this part are still in en-US).